docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -319,6 +398,29 @@ updated state of the game and store it back at the address from which it came.
       })
     }
 
+{% elif language == 'Java' %}
+
+.. code-block:: java
+
+    private void storeGameData(String gameName, GameData gameData, State stateStore) {
+      String address = makeGameAddress(gameName);
+
+      String encodedGameData = encodeData(gameData)
+      ByteString gameByteString = ByteString.copyFromUtf8(encodedGameData);
+
+      Map.Entry<String, ByteString> entry
+          = new AbstractMap.SimpleEntry<>(address, gameByteString);
+
+      Collection<Map.Entry<String, ByteString>> addressValues
+          = Collections.singletonList(entry);
+
+      Collection<String> addresses = stateStore.set(addressValues);
+
+      if (addresses.size() < 1) {
+        throw new InternalError("State Error");
```

Comment:
Realizing that this might be a bug in the transaction family itself:  this should probably be an invalid transaction, as it most likely means that that inputs and outputs for the transaction don't match the generated transaction.